docs/architecture/porting-existing-aspnet-apps/deployment-scenarios.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Deployment scenarios when migrating to ASP.NET Core
+description: 
+author: ardalis
+ms.date: 11/13/2020
+---
+
+# Deployment scenarios when migrating to ASP.NET Core
+
+[!INCLUDE [book-preview](../../../includes/book-preview.md)]
+
+*Notes*
+
+- Focus on IIS for ASP.NET
+- Show strategies, especially for web api, for using IIS routing to enable side by side deployments (be sure to reference this early on in book as well)
+
+"Deployment may make sense sprinkled in throughout the book. Kind of like how we’ve discussed how customers should think about branching, feature spikes, etc and when they can have decent checkpoints."
```

Comment:
```suggestion
"Deployment may make sense sprinkled in throughout the book. Kind of like how we've discussed how customers should think about branching, feature spikes, and when they can have decent checkpoints."
```

Reply:
This one is literally just notes that won't be in the final text but sure I'll take the change...